components/inputs/test/input-select.visual-diff.js

```diff
@@ -0,0 +1,39 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
+
+describe('d2l-input-select', () => {
+
+	const visualDiff = new VisualDiff('input-select', __dirname);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await browser.newPage();
+		await page.setViewport({width: 800, height: 800, deviceScaleFactor: 2});
+		await page.goto(`${visualDiff.getBaseUrl()}/components/inputs/test/input-select.visual-diff.html`, {waitUntil: ['networkidle0', 'load']});
+		await page.bringToFront();
+	});
+
+	after(() => browser.close());
+
+	beforeEach(async() => {
+		await visualDiff.resetFocus(page);
+	});
+
+	['default', 'overflow', 'disabled', 'invalid', 'rtl', 'rtl-overflow'].forEach((name) => {
+		it(name, async function() {
+			const rect = await visualDiff.getRect(page, `#${name}`);
+			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+		});
+	});
+
+	['default', 'overflow', 'invalid', 'rtl'].forEach((name) => {
```

Comment:
no `rtl-overflow` here?

Reply:
I wasn't too worried about it, but seeing as the new design calls for some gradients maybe I'll add it...